src/main/java/org/verdictdb/coordinator/QueryResultAccuracyEstimatorFromDifference.java

```diff
@@ -7,26 +7,25 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
-public class AsyncAggExecutionRulerByConverge extends AsyncAggExecutionRuler {
+public class QueryResultAccuracyEstimatorFromDifference extends QueryResultAccuracyEstimator {
 
   VerdictResultStream resultStream;
 
   Coordinator runningCoordinator;
 
-  // the values of the result should be within [(1-valueError), (1+valueError)] of the previous result.
-  // Otherwise, it is not converged.
+  // the values of the result should be within [(1-valueError)*prevValue, (1+valueError)*prevValue] of the previous result.
+  // Otherwise, it will fetch next result.
   Double valueError = 0.05;
 
-  // the #column of the result should be within [(1-columnError), (1+columnError)] of the previous result.
-  // Otherwise, it is not converged.
-  Double columnError = 0.05;
+  // the #column of the result should be within  [(1-groupCountError)*prev#column, (1+groupCountError)*prev#column] of the previous result.
+  // Otherwise, it will fetch next result.
+  Double groupCountError = 0.05;
 
   // key is the values of non-aggregated column, value is the values of aggregated column
   HashMap<List<Object>, List<Object>> aggregatedMap = new HashMap<>();
 
-  AsyncAggExecutionRulerByConverge(VerdictResultStream resultStream, Coordinator runningCoordinator) {
+  QueryResultAccuracyEstimatorFromDifference(VerdictResultStream resultStream, Coordinator runningCoordinator) {
```

Comment:
Please remove `resultStream` for the argument list of this constructor. Just let the caller of this class rely on the add() method.